src/detect-byte-extract.c

```diff
@@ -409,7 +423,12 @@ static inline DetectByteExtractData *DetectByteExtractParse(const char *arg)
                            "for arg %d in byte_extract", i);
                 goto error;
             }
-            bed->align_value = atoi(align_str);
+            if (ByteExtractStringUint8(&bed->align_value, 10, 0,
+                                       (const char *)align_str) < 0) {
+                SCLogError(SC_ERR_INVALID_SIGNATURE, "Invalid value for"
+                        " align_str: \"%s\".", align_str);
```

Comment:
align_value rather than align_value ?

Reply:
Yes, this should be align_value. Fixing in the next PR.